### PR TITLE
Return JSON entity for content-type "application/json".

### PR DESCRIPTION
**Pull request by chrisr3**
We claim that the content type is "application/json" so return a JSON entity.
Also use HTTP status 200 because the endpoint has completed successfully.

### CODE REVIEW

File: java-source/src/main/java/com/template/api/TemplateApi.java [hunk 1]
```diff
@@ -12,6 +14,7 @@
 // This API is accessible from /api/template. The endpoint paths specified below are relative to it.
 @Path("template")
 public class TemplateApi {
+    private final ObjectMapper mapper = new ObjectMapper();
```
Comment (Clintonio) [inline java-source/src/main/java/com/template/api/TemplateApi.java]: While this is one option the default mapper we have applied allows you to simply do:

     Response.ok(mapOf("name" to "Template GET endpoint")).build()

To get the same effect. The same is true of the other endpoint.

Comment (Clintonio) [inline java-source/src/main/java/com/template/api/TemplateApi.java]: On top of that a single quoted string is valid JSON:

http://stackoverflow.com/questions/13318420/is-a-single-string-value-considered-valid-json

Though the change is reasonable if you use mapOf and change the keyname to "message" or something.

Comment (Clintonio) [inline java-source/src/main/java/com/template/api/TemplateApi.java]: Hah, fair. But can you use the existing mapper as I detailed in my first response?

**Comment by chrisr3** [comment]
Back to you.

**Pull request merged**